sailor/assetcentral/system.py

```diff
@@ -5,26 +5,27 @@
 """
 from __future__ import annotations
 
+import math
 import itertools
 import logging
-from typing import TYPE_CHECKING, Union
+from typing import Union
 from datetime import datetime
 from functools import cached_property
 from operator import itemgetter
 
 import pandas as pd
 
-from sailor import _base
-from sailor import sap_iot
+from sailor import _base, sap_iot
+import sailor.assetcentral.indicators as ac_indicators
```

Comment:
this import seems unnecessary.

you can import the  `AggregatedIndicatorSet` directly in line 24 where you import other classes from `indicators`